Tuple4.elm

```diff
@@ -114,20 +116,18 @@ sortWith cmp ( a, b, c, d ) =
     let
         goesBefore x y =
             not <| cmp x y == GT
+
+        ( a_, b_, c_ ) =
+            Tuple3.sortWith cmp ( a, b, c )
```

Comment:
Though you've mentioned fixing the function. Does that mean there were inputs it misbehaved for? If yes, then adding an explicit, non-fuzzed test case for that would be nice.